examples/suggest-psp/README.md

```diff
@@ -0,0 +1,34 @@
+# Suggest Pod Security Policy
+
+The `suggest-psp` KRM config function lints pod security policies by suggesting
+the 'spec.allowPrivilegeEscalation' field be set to 'false'. It outputs
+structured results detailing which PSP objects should be changed. This example
+invokes the suggest-psp function using declarative configuration.
+
+## Function Invocation
+
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/suggest-psp .
+kpt fn run suggest-psp --results-dir /tmp
+```
+
+## Expected Results
+
+The `--results-dir` flag let us specify a destination to write function results
```

Comment:
Remove `--results-dir` flow if not needed to demonstrate this.

Reply:
It's needed to show that the issue is with the `spec.allowPrivilegeEscalation` field and that its `suggestedValue` is `false`.